ChangeLog.md

```diff
@@ -255,7 +257,9 @@ See docs/process.md for more on how version tagging works.
   (#18861)
 - The `emscripten_proxy_async_with_callback` API was replaced with a simpler
   `emscripten_proxy_callback` API that takes a second callback to be called if
-  the worker thread dies before completing the proxied work.  
+  the worker thread dies before completing the proxied work.
+
+  <p align="right">(<a href="#top">🔼 Back to top</a>)</p>
```

Comment:
I'd rather no embed html stuff into this .md file.. but maybe others feel differently?   

Where are you reading that this useful?  In the github UI?  Does the home key not work for that?